north/Cargo.toml

```diff
@@ -45,6 +45,7 @@ toml = { version = "0.5.6", optional = true }
 url = { version = "2.1.1", optional = true }
 uuid = { version = "0.8.1", features = [ "v4"], optional = true }
 zip = { version = "0.5.6", optional = true }
+ctrlc = { version = "3.1.6", features = ["termination"] }
```

Comment:
nit: sort dependencies alphanumerically

Why is this a feature? Think we can have this all the time.

Reply:
This  feature makes the handle catch `SIGTERM` in addition to `SIGINT`.